plugins/polio/export_utils.py

```diff
@@ -0,0 +1,114 @@
+from ctypes import alignment
+from curses import start_color
+from turtle import color
+from openpyxl import Workbook
+from openpyxl.writer.excel import save_virtual_workbook
+from openpyxl.utils import get_column_letter
+from openpyxl.styles import Alignment, Font, PatternFill
+from openpyxl.styles.borders import Border, Side
+import datetime as dt
+import calendar
+
+# from datetime import datetime
+
+
+def generate_xlsx(filename, datas):
```

Comment:
General remark (not especially for this function): ideally, it would be great to have 1) a docstring 2) some type annotations (don't worry if you need help for this, we can work on that during daily 1-1)